lambdas/commands.py

```diff
@@ -20,32 +20,47 @@ def generate_extract_command(
         source, run_date, run_type, step
     )
     extract_output_file = helpers.generate_step_output_filename(
-        "index", extract_output_prefix, step
+        source, "index", extract_output_prefix, step
     )
 
-    extract_command = [
-        f"--host={input_data['oai-pmh-host']}",
-        f"--output-file=s3://{timdex_bucket}/{extract_output_file}",
-    ]
+    extract_command = []
 
     if verbose:
         extract_command.append("--verbose")
 
-    extract_command.append("harvest")
-    extract_command.append(f"--metadata-format={input_data['oai-metadata-format']}")
-
-    if source in ["aspace", "dspace"]:
-        extract_command.append("--method=get")
+    if source in ["gismit", "gisogm"]:
```

Comment:
@jonavellecuerdo - I'm sorry I didn't propose this the first time around, but seeing it on a 2nd pass what do you think about putting `["gismit", "gisogm"]` into a config level constant?  Then we could refer to that constant (list) throughout the code; counting at least 3-4 instances where we check a variation of the source name being one of those two values.

Reply:
Definitely agree this is a good change to make!